apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/nodes/NodesWebscript1.java

```diff
@@ -482,6 +486,57 @@ public void getAllInfoOfNodes(WebScriptRequest request, WebScriptResponse respon
         //logger.error("end writeJsonResponse");
     }
 
+    @ApiOperation(value = "Retrieves the parents of the nodes recursively",
+            notes = "example json request data: \n"
+                    + "```\n"
+                    + "{\n"
+                    + "\t\"rootRef\" : \"workspace://SpacesStore/c5a7fe48-d4ab-401c-bf38-e51a297f572f\",\n"
+                    + "\t\"nodeRefs\" : [\n"
+                    + "\t\t\"workspace://SpacesStore/f3e914be-c112-45b4-a398-6ceb6bd19958\",\n"
+                    + "\t\t\"workspace://SpacesStore/74d2de42-c03d-410b-bfc5-12ab487840b8\"\n"
+                    + "\t\t]\n"
+                    + "}\n"
+                    + "```\n"
+                    + "\n"
+                    + "Parameter 'rootRef' is an optional parameter. If the parameter is not given the default value\n"
+                    + "will be the node reference of Company Home. It is the last parent last parent in the list of\n"
+                    + "recursive parents."
+                    + "Parameter 'nodeRefs' is a mandatory parameter. It contains the node references of the nodes\n"
+                    + "for which the recursive parents need to be retrieved.\n"
+                    + "\n"
+                    + "Nodes for which the recursive parents are requested are omitted from the result if the node\n"
+                    + "does not exist or one of its recursive parents cannot be read by the user.")
+    @Uri(value = "/nodes/parent/recursive", method = HttpMethod.POST)
```

Comment:
As discussed over Slack, I would like a single-object operation via HTTP `GET`. The overhead of multiple calls can be avoided via the bulk call .